pxr/imaging/lib/glf/imageRegistry.cpp

```diff
@@ -68,7 +68,8 @@ GlfImageRegistry::_ConstructImage(std::string const & filename)
     static GlfImageSharedPtr NULL_IMAGE;
 
     // Lookup the plug-in type name based on the filename.
-    TfToken fileExtension(ArGetResolver().GetExtension(filename));
+    TfToken fileExtension(
+            TfStringToLower(ArGetResolver().GetExtension(filename)));
```

Comment:
It would be better to not change the case when printing the unknown image type error message. You could put to tolower in the call to Find(), or you could change the error message to print the entire filename (which is what the Python version is doing).